### PR TITLE
Skip containerd shutdown if Docker is bound to it

### PR DESCRIPTION
**Pull request by tstromberg**
A simple approach to something Medya was discussing for #7435

In the KIC configuration, Docker is bound to containerd.

### CODE REVIEW

File: pkg/minikube/cruntime/cruntime.go [hunk 1]
```diff
@@ -178,13 +179,22 @@ func disableOthers(me Manager, cr CommandRunner) error {
 		if r.Name() == me.Name() {
 			continue
 		}
+
+		// Don't disable containerd if we are bound to it
+		if me.Name() == "Docker" && r.Name() == "containerd" && dockerBoundToContainerd(cr) {
```
Comment (medyagh) [inline pkg/minikube/cruntime/cruntime.go:184]: is "Docker" capitalized ?

Comment (tstromberg) [reply pkg/minikube/cruntime/cruntime.go:184]: Yes. It's the string we output to console :(

**Comment by medyagh** [comment]
the soft start on this PR is 13 seconds ( on my PR it was 8 seconds)
https://github.com/kubernetes/minikube/pull/7435

could be I am using different laptop ( but maybe somethign is wrong?)
```
medya@~/workspace/minikube (tstromberg-containerd-bound) $ time ./out/minikube start --driver=docker
😄  minikube v1.9.2 on Darwin 10.15.3
✨  Using the docker driver based on existing profile
👍  Starting control plane node minikube in cluster minikube
🚜  Pulling base image ...
🏃  Updating the running docker "minikube" container ...
🐳  Preparing Kubernetes v1.18.0 on Docker 19.03.2 ...
    ▪ kubeadm.pod-network-cidr=10.244.0.0/16
🌟  Enabling addons: default-storageclass, storage-provisioner
🏄  Done! kubectl is now configured to use "minikube"

real	0m13.866s
user	0m2.883s
sys	0m1.686s
```

**Comment by tstromberg** [comment]
@medyagh - This is not expected to have a major impact on soft start speed. You can however confirm using `--alsologtostderr` that it does not shutdown containerd.

**Pull request merged**